src/components/new-safe/CreateSafe/index.tsx

```diff
@@ -56,15 +57,17 @@ const staticHints: Record<
         title: 'Safe Setup',
         text: (
           <>
-            Not sure how many owners and confirmations you need for your Safe?{' '}
-            <Link
-              sx={{ textDecoration: 'underline' }}
+            Not sure how many owners and confirmations you need for your Safe?
+            <br />
+            <b>Learn more about setting up your Safe.</b>
```

Comment:
I would include this inside the link. What do you think?

Reply:
Makes sense!